src/pykoala/data_container.py

```diff
@@ -361,58 +389,157 @@ class DataContainer(object):
 
     Attributes
     ----------
-    intensity
-    variance
-    intensity_units
-    info
-    log
-    mask
-    mask_map
+    intensity : astropy.Quantity
+        Array with the counts/surface brightness/... at each pixel.
+    variance : astropy.Quantity
+        Uncertainties associated to the `intensity` values.
+    mask : DataMask
+        Pixel flags.
+    info : dict
+        Parameters describing the data.
+    log : HistoryLog
+        History log reporting the data reduction steps undertaken so far.
 
     Methods
     -------
     # TODO
     """
 
+    @property
+    def intensity(self):
+        """Array with the counts/surface brightness/... at each pixel."""
+        return self._intensity
+
+    @intensity.setter
+    def intensity(self, value):
+        self._intensity = value
+
+    @intensity.deleter
+    def intensity(self):
+        del self._intensity
```

Comment:
I guess it's fine, but do we need to include deleters? In which situation are they going to be used?